src/traces/pie/index.js

```diff
@@ -373,6 +373,12 @@ pie.plot = function(gd, cdpie) {
                     }
                 }
 
+                function handleClick (evt) {
+                    gd._hoverdata = pt;
```

Comment:
It's an object. We could wrap it in an array so that `points` that is returned with the `plotly_click` event matches scatter events, but it doesn't make sense in the context - there will only be element ever passed - the slice that's clicked on.